packages/papaya/backend/sirius-components-papaya/src/main/resources/model/papaya.ecore

```diff
@@ -16,6 +16,11 @@
   <eClassifiers xsi:type="ecore:EClass" name="Project" eSuperTypes="#//NamedElement">
     <eStructuralFeatures xsi:type="ecore:EReference" name="projects" upperBound="-1"
         eType="#//Project" containment="true"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="contractualStartDate" eType="#//LocalDate"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="dayDuration" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EIntegerObject"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="contractualEndDate" eType="#//LocalDate"
+        changeable="false" volatile="true" transient="true" derived="true"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="isSensitive" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EBooleanObject"/>
```

Comment:
I considered that papaya could be changed as long as it increases the complexity.
In my case, contractualStartDate and contractualEndDate allow to display simple dates in detail view
dayDuration and isSensitive are used to show that EIntegerObject and EBooleanObject are supported in default Details with slightly different behavior.

Anyway, since your commit af709c59 default Detail view can not be used anymore so this changes are not useful anymore for my purpose.
I propose to move `Iteration.startDate` and `Iteration.endDate` type from Instant to LocalDate. This way we would have an example of metamodel use LocalDate
Task.isDone is already passed from EBoolean to EBooleanObject

Reply:
done

Reply:
done

Reply:
> I considered that papaya could be changed as long as it increases the complexity.

No.

I have created Papaya as a tool to help me document Sirius Web with Sirius Web, as a side effect, it also helps me challenge the work done by everybody in Sirius Web. Consider it as my tool for end to end test of Sirius Web. If an improvement does not help me document Sirius Web with Sirius Web, it's not relevant to my needs. Those changes to the Papaya metamodel do not bring anything for me regarding this goal.

- The fields `Task#income` or `Task#funding` have not meaning whatsoever for Sirius Web.
- Defining those fields in an inefficient manner with `EIntegerObject` (and changing existing fields to use `EIntegerObject` or `EBooleanObject`) does not help me at all to document Sirius Web. I would consider that minor regressions in the usability of the metamodel instead of any kind of improvement.
- Those new fields are not used at all to document Sirius Web. If at least they were used in the creation of the default model or if there was a credible plan to use them soon, I would understand but here it's just using my metamodel as a shortcut to create tests.

Updating Papaya is not to be used as a convenient way to create new tests. It would end up being a mess very quickly.

Reply:
It's OK, the changes made to papaya were removed from the PR